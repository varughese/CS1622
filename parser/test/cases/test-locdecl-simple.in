void f0(void)
{
    /* local-declarations with one item */
    int x;
}

void f1(void)
{
    /* local-declarations with three items */
    int x;
    int y;
    int z[10];
}

void f2(void)
{
    /* empty local-declarations */
}

void main(void)
{
}
