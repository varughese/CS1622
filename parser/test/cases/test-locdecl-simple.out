[program
  [fun-declaration
    [void]
    [f0]
    [params]
    [compound-stmt 
      [var-declaration [int] [x]]
    ]
  ]
  [fun-declaration
    [void]
    [f1]
    [params]
    [compound-stmt 
      [var-declaration [int] [x]]
      [var-declaration [int] [y]]
      [var-declaration [int] [z] [10]]
    ]
  ]
  [fun-declaration
    [void]
    [f2]
    [params]
    [compound-stmt 
    ]
  ]
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
    ]
  ]
]
